.github/workflows/ci.yml

```diff
@@ -48,7 +48,20 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - uses: astral-sh/setup-uv@v5
       - run: uv tool install poethepoet
-      - run: uv sync --all-extras
+      - run: uv sync --all-extras --python ${{ matrix.python }}
```

Comment:
What is the use case for `--python` instead of letting it use the existing installed one? (even if we change the install method)